package.json

```diff
@@ -19,9 +19,11 @@
   "scripts": {
     "postinstall": "cd lib || builder run npm:postinstall",
```

Comment:
This is where the git install magic happens. When installing from the npm repo, you will have a `lib` folder, so the build gets skipped. When installing from git, there is no `lib` folder so it kicks off the build process.